pkg/ccl/backupccl/backup_processor.go

```diff
@@ -613,9 +614,11 @@ func runBackupProcessor(
 								span.lastTried = timeutil.Now()
 								span.attempts++
 								log.VEventf(ctx, 1, "retrying ExportRequest for span %s; encountered WriteIntentError: %s", span.span, lockErr.Error())
-								// If we're not mid-key we can put this on the the queue to give
-								// it time to resolve on its own while we work on other spans.
-								if span.firstKeyTS.IsEmpty() {
```

Comment:
to what end?

Reply:
In general, It seems like a good idea to put  a resume span that encountered a writeIntentError at the back of the queue, to allow intent resolution time to do its thing.  I think we should only stray from  this behavior if we absolutely need to, i.e. only if the resume span is split mid row.

Reply:
I think @stevendanna was suggesting we get rid of the "to be back of the queue" behavior _altogether_ actually, mid-row or otherwise, since it forces out-of-order flushes, and only really matter in the first few seconds of a backup. Once we're reading far enough in the past we can just use priority and get it done now. The closed timestamp system already means that there is a 6s clock to commit, so this "I'll leave it for now and maybe it'll commit before I come back around" behavior is pointless anyway.

Reply:
huh, i suppose we should rip it out then.